R/pstr_prepare_scenario.R

```diff
@@ -19,6 +19,15 @@
 #'
 #' pstr_prepare_scenario(raw_scenarios)
 pstr_prepare_scenario <- function(scenarios) {
+  # Hack #308
+  if (hasName(scenarios, "weo")) {
+    scenarios$weo <- rename(
+      scenarios$weo,
+      weo_sector = "weo_product",
+      weo_subsector = "weo_flow"
+    )
+  }
+
   imap_dfr(scenarios, ~ pstr_prepare_scenario_impl(.x, .y))
 }
```

Comment:
We now rename inside `pstr_prepare_scenario()`.

Reply:
@maurolepore Thanks! It makes more sense to include it in `prepare` function.